dotcom-rendering/src/components/TableOfContents.importable.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions -- TODO */
+/* eslint-disable jsx-a11y/click-events-have-key-events -- TODO */
```

Comment:
Do you have any more information on why disabling these is necessary? Is it something we could capture in an issue to be fixed in a follow-up?

Reply:
I've updated @guardian/tsconfig which i assume has tightened the rules, we should add a ticket to resolve them though yes.

Reply:
https://github.com/guardian/dotcom-rendering/issues/8161